ChangeLog.md

```diff
@@ -21,6 +21,8 @@ See docs/process.md for more on how version tagging works.
 3.1.52 (in development)
 -----------------------
 - The `--default-obj-ext` command line flag was removed. (#20917)
+- Remove JAVA from the list of `.emscripten` config file settings.  In the
```

Comment:
Can you add here what you wrote in the commit message (i.e. how a user who was previously using this feature can do it via EMCC_CLOSURE_ARGS)?